aio/tools/transforms/angular-api-package/processors/mergeOverriddenImplementation.js

```diff
@@ -78,7 +77,7 @@
           });
 
           // Copy the constructor overrides from the constructor doc, if any are present.
-          if (!ctorDoc.members || ctorDoc.members.length !== 1 || !ctorDoc.members[0].name.includes('new')) {
+          if (!ctorDoc.members || ctorDoc.members.length === 0 || !ctorDoc.members[0].name.includes('new')) {
             throw new Error(`@overriddenImplementation requires that the provided constructor ${symbolNames[1]} have exactly one member called "new", possibly with multiple overrides.`);
```

Comment:
It sounds like it is no longer a requirement to have exactly one member.

```suggestion
            throw new Error(`@overriddenImplementation requires that the provided constructor ${symbolNames[1]} have a member called "new", possibly with multiple overrides.`);
```

Reply:
Right, good catch. This changed due to the need to provide the explicit prototype property for google3 backwards compatibility.